src/tests/posix/pthread/pthread_kill_test.c

```diff
@@ -0,0 +1,121 @@
+/**
+ * @file
+ * @brief test for pthread_kill
+ *
+ * @author  XU Chenglin
+ * @date    09/03/2020
+ */
+
+#include <pthread.h>
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+#include <signal.h>
+#include <embox/test.h>
+
+
+EMBOX_TEST_SUITE("posix/pthread_kill");
+
+
+void sighand(int signo)
+{
+    return;
+}
+
+void *thread_test(void *parm)
+{
+    int rc = sleep(10);
```

Comment:
sleep 10 sec? Is not too long?

Reply:
How about 5 or 3? How many second is better?